### PR TITLE
I have added a list of users.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**
* I have added a list of users, from where we can go to the correspondence section of each.
* In all components that receive props, I added props types.

### CODE REVIEW

File: navigation/BottomTabNavigation.jsx [hunk 1]
```diff
@@ -9,7 +8,8 @@ import ProfileIcon from '../assets/icons/Profile.svg';
 import MessagesIcon from '../assets/icons/Messages.svg';
 
 import StackNavigation from './StackNavigation';
-import MessagesList from '../screens/Messages/MessagesList';
+// import MessagesList from '../screens/Messages/MessagesList';
```
Comment (M7Arman) [inline navigation/BottomTabNavigation.jsx:11]: remove comment if you don't need it. If you need add some explanation as a c omment

File: screens/Content/ContentScreen.jsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import {useContext} from 'react';
+import React, {useContext} from 'react';
```
Comment (M7Arman) [inline screens/Content/ContentScreen.jsx:1]: why did you add Reacy? do you really need it?

File: screens/Content/ContentScreen.jsx [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+import PropTypes from 'prop-types';
 import {TextInput, View} from 'react-native';
 import {SwiperFlatList} from 'react-native-swiper-flatlist';
 import {DNAdataContext} from '../../Data/data';
-import {renderItem} from './renderItem';
+import renderItem from './renderItem';
```
Comment (M7Arman) [inline screens/Content/ContentScreen.jsx:6]: don't use render word in the component name, try to find a better name that will explain what is inside the component

File: navigation/BottomTabNavigation.jsx (hunk 1 above)
Comment (MiqayelChaloyan) [reply navigation/BottomTabNavigation.jsx:11]: Removed comment.

File: screens/Content/ContentScreen.jsx (hunk 3 above)
Comment (MiqayelChaloyan) [reply screens/Content/ContentScreen.jsx]: Changed component name
